client/components/tabs/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
```

Comment:
Instead of Tabs, how About TabBar since this only renders a Tab Bar and not the Tab content.